Header header
geometry_msgs/Pose2D pose
string id